packages/kbn-securitysolution-list-utils/src/helpers/index.ts

```diff
@@ -95,6 +95,10 @@ export const filterExceptionItems = (
         }
       }, []);
 
+      if (entries.length === 0) {
```

Comment:
Did something change with the schema? Shouldn't line 100/102 catch this since entries was previously typed as not being able to be empty?

Reply:
I think something was broken during the refactor or lift and shift because I remember having logic in the filter function at some point protecting this. As for the schema, I don't know if we ever had a non-empty requirement, only that it needs to be an array of `BuilderEntry` and it's possible for that to just be an empty array. Either way it's causing bugs now and I thought it'd be better to fix it in the package itself than to fix it in the `add_exception_modal` file.

Reply:
hmmm ok. I could've sworn we had it typed to be non-empty at some point. I just wanted to make sure we were addressing the issue at the right place. LGTM-ed.